Exiled.API/Features/Player.cs

```diff
@@ -1035,9 +1041,9 @@ public void ShowHint(string message,float duration = 3f)
                 new OutlineEffect(new Color32(0, 0, 0, 0), 5f, 0f, 1f),
```

Comment:
Can these arguments be personalized? If so, adding a "Color32" and "outlineWidth" as optional parameters would be cool.

Reply:
I just checked to see if OutlineEffect actually effected the hint but it did not so it will be removed